downstream/modules/eda/proc-eda-set-up-token.adoc

```diff
@@ -24,7 +24,7 @@ For more information about creating the token, see the link:https://docs.ansible
 +
 [NOTE]
 ====
-The token must be in write-scope.
+Your token must be in write-scope and if you are the assigned user, you must have permission to execute job and workflow templates.
 ====
```

Comment:
I think you need a + after the closing ==== for the note to make sure the numbering is continued but I'm not %100 certain.

Reply:
Ignore this. I built it locally and it formats ok. I'll proceed with merging.